fastlane/metadata/ru/name.txt

```diff
@@ -0,0 +1 @@
+WordPress – конструктор сайтов
```

Comment:
Im guessing you got that infamous `429 Too Many Requests` on `ru` during your previous run which is why it gets reintroduced now? (As I'm sure the metadata were already previously translated in Russian in GlotPress and that translation of name+keywords+description+subtitle in Russian isn't brand new?)

Reply:
Damn 🤦‍♂️ 

I've taken the habit of tracking the 429 occurrences, but the code freeze day for 20.1 came with a lot of other things and I didn't not do it. That's to say, I think your guess is right, but I can't prove it.